frontend/src/components/common/Filter/common/utils.ts

```diff
@@ -22,3 +47,42 @@ export function ontologyCellAccessorFn(
   return (categories: Categories) =>
     categories[categoryKey].map((o: Ontology) => o.label);
 }
+
+/**
+ * Format number to the correct scale and possibly round.
+ * @param num - Number to format.
+ * @returns String representation of given number, possibly rounded, with corresponding scale symbol.
+ */
+export function formatNumberToScale(num: number): string {
+  // Return numbers less than 1000 rounded.
+  if (num < 1000) {
```

Comment:
should we use the constants here?

E.g., `num < SCALE_THOUSAND`, `num < 10 * SCALE_THOUSAND`, `num < SCALE_MILLION`, etc.